src/libmongoc/src/mongoc/mongoc-server-description-private.h

```diff
@@ -184,11 +194,10 @@ void
 mongoc_server_description_set_topology_version (mongoc_server_description_t *sd,
                                                 const bson_t *tv);
 
-/* If a service_id is set on the topology description, copies it to @oid and
- * returns true. Otherwise returns false and zeros out oid.
- */
+extern const bson_oid_t kZeroServiceId;
+
 bool
-mongoc_server_description_service_id (
-   const mongoc_server_description_t *description, bson_oid_t *oid);
+mongoc_server_description_has_service_id (
```

Comment:
This is now more consistent with the existing `mongoc_server_description_has_election_id`, which is another ObjectID field on the server description.